api/app/channels/trust_monitor/publisher.py

```
@@ -34,41 +36,97 @@ def __init__(
+        requested = False
+        scheduled = True
         if surface in {TrustAlertSurface.ADMIN_UI, TrustAlertSurface.BOTH}:
+            requested = True
+            # TrustMonitorService durably stores the finding before publishing;
+            # that stored row is the ADMIN_UI delivery surface. An optional
+            # admin_publisher is only an additional side effect.
             if self.admin_publisher is not None:
                 try:
-                    self.admin_publisher.publish(finding)
+                    scheduled = self.admin_publisher.publish(finding) and scheduled
                 except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)